packages/cli/src/index.ts

```diff
@@ -33,4 +30,16 @@ program
     newSpace(options.key, options.input, options.name, options.debug);
   });
 
+program
+  .command('integrate')
+  .description('integrate Builder.io with an existing codebase, currently supports Next.js')
+  .option('-d,--debug', 'print debugging information')
+  .option('-s,--stack <stack>', 'currently supports nextjs', 'nextjs')
+  .option('-m,--model <model>', 'name of the model you want to integrate')
+  .option('-a,--apiKey <apiKey>', 'you can find your apiKey on builder.io/account/settings')
+  .option('-p,--pathPrefix <prefix>', 'ADD SOMETHING GOOD HERE', '/m/')
```

Comment:
I think you're missing the `language` flag you just added!

I would make sure that it's optional btw, and default to JS instead of requiring a value.

Reply:
Looks from the code that you did make it optional, so no worries there.